website/docs/expansion.md

```diff
@@ -0,0 +1,223 @@
+---
+id: expansion
+title: Early Rejection of Generator Resources
```

Comment:
Since this works with `dryrun` or `warn` too, do we want to call this rejection?

Reply:
I think you had mentioned "Early Validation of Generator Resources"

Reply:
PSA called it `workload object` and `workload resources`
https://kubernetes.io/docs/concepts/security/pod-security-admission/#workload-resources-and-pod-templates
How about `Early Validation of Workload Resources`

Reply:
or just `Validation of Workload resources`?

Reply:
wdyt @davis-haba @maxsmythe

Reply:
SGTM, the concept is more general that just "workload", but I suspect that'll be the most common touch point.